contributing/generate-readmes.py

```diff
@@ -56,15 +56,15 @@ def generate_main_readme(directory_of_this_script, chapters, chapter_titles,
     readme = directory_of_this_script / '..' / 'README.md'
     with open(readme, 'w') as f:
         f.write("""
-[![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://raw.githubusercontent.com/bast/cmake-recipes/master/LICENSE)
+[![License: MIT](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://raw.githubusercontent.com/dev-cafe/cmake-cookbook/master/LICENSE)
 
-[![Travis branch](https://img.shields.io/travis/bast/cmake-recipes/master.svg?style=flat-square)](https://travis-ci.org/bast/cmake-recipes)
-[![AppVeyor branch](https://img.shields.io/appveyor/ci/bast/cmake-recipes/master.svg?style=flat-square)](https://ci.appveyor.com/project/bast/cmake-recipes/branch/master)
-[![Drone branch](https://www.drone-ci.science/api/badges/bast/cmake-recipes/status.svg?style=flat-square)](https://www.drone-ci.science/bast/cmake-recipes)
+[![Travis branch](https://img.shields.io/travis/dev-cafe/cmake-cookbook/master.svg?style=flat-square)](https://travis-ci.org/dev-cafe/cmake-cookbook)
+[![AppVeyor branch](https://img.shields.io/appveyor/ci/dev-cafe/cmake-cookbook/master.svg?style=flat-square)](https://ci.appveyor.com/project/dev-cafe/cmake-cookbook/branch/master)
+[![Drone branch](https://www.drone-ci.science/api/badges/dev-cafe/cmake-cookbook/status.svg?style=flat-square)](https://www.drone-ci.science/dev-cafe/cmake-cookbook)
```

Comment:
The drone instance has to be nuked and set up from scratch. I'd remove this mention of drone and adjust later.

Reply:
OK - sorry for extra work. I will remove that line from the README.